sygnal/webpushpushkin.py

```diff
@@ -272,3 +296,6 @@ class HttpResponseWrapper:
 
     def __init__(self, deferred):
         self.deferred = deferred
+
+    async def read_body(self, response):
+        return (await readBody(response)).decode()
```

Comment:
I'm not sure this is really useful to have in the wrapper since `pywebpush` doesn't expect this? It seems to be conflating concerns. Can we inline this code above?